google/cloud/spanner_dbapi/connection.py

```diff
@@ -15,13 +15,24 @@
 
 
 class Connection:
+    """A wrap-around object for the existing :class:`Database` and the corresponding :class:`Instance` objects
+
+    :type is_closed: bool
+    :param is_closed: (Optional) Flag of the connection state (open/closed). Default is False.
+    """
```

Comment:
`is_closed` is a property, we don't describe properties in a class definition, we describe `__init__` args there